### DISCUSSION TITLE
Export to ods

### DISCUSSION THREAD

**Discussion by engahlw** [manuvarkey/GElectrical#15] [Q&A]
Is there a possibility to export the reports directly in ods format?

**Comment by manuvarkey** [comment]
Currently this is not implemented. Which portions of the report are required ?

It will be possible to implement `HTML` export of report since internally, the `PDF` file is converted from an `HTML` file. Tables from the `HTML` file can be easily added to `ods` document.

**Comment by manuvarkey** [comment]
Sample report

[report.zip](https://github.com/manuvarkey/GElectrical/files/14109722/report.zip)
